packages/react-google-maps-api/.eslintrc.js

```diff
@@ -33,8 +34,8 @@ module.exports = {
     'standard',
     'no-inferred-method-name',
     'react-functional-set-state',
-    '@typescript-eslint',
-    'prettier'
+    '@typescript-eslint'
+    // 'prettier'
```

Comment:
I'll open an issue regarding prettier and linting in general because I see we're spending too much of our time over conflicting rules and setups

Reply:
I found that I can wrap code instead of horizontal scrolling, I can live with it.